proto-google-cloud-datastore-v1/pom.xml

```diff
@@ -24,6 +24,16 @@
       <scope>compile</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>flatten-maven-plugin</artifactId>
+      </plugin>
+    </plugins>
+  </build>
+
   <profiles>
     <profile>
```

Comment:
Can you add the version tag to each of the java9 profiles in the repo to avoid the bug we ran into yesterday with java 9+?

Reply:
Done.